src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -540,6 +544,11 @@ async Task<bool> OnPause(SessionId sessionId, JObject args, CancellationToken to
 
                         var method = asm.GetMethodByToken(method_token);
 
+                        if (method  == null && !asm.Image.HasSymbols)
+                        {
+                            method = await LoadSymbolsOnDemand(asm, method_token, sessionId, token);
```

Comment:
Do we need a try/catch around this, since it's optional symbols? Maybe even move the one from `LoadSymbolsOnDemand`, to here.